static/css/base/helpers-misc.less

```diff
@@ -128,6 +128,9 @@ button {
 .neg,
 .red {
   color: @red !important;
+  &:empty {
+    display: none;
+  }
```

Comment:
Is there a better way to do this?  I believe that this is the root cause of #6933

Reply:
Fixed; good find on the root cause!